implementations.md

```diff
@@ -17,5 +17,6 @@ Projects or Companies currently adopting the OCI Image Specification
 * [coreos/rkt](https://github.com/coreos/rkt)
 * [box-builder/box](https://github.com/box-builder/box)
 * [coolljt0725/docker2oci](https://github.com/coolljt0725/docker2oci)
+* [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/image-formats)
```

Comment:
GAR seems to be replacing GCR now https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr

Reply:
Also wondering the list of registry services should be rather maintained the dist spec repo

Reply:
Makes sense, we can have GCR and ECR submit to https://github.com/opencontainers/oci-conformance rather than listing here. The registries support distribution-spec more than image-spec.